src/diem/testing/miniwallet/config.py

```diff
@@ -10,14 +10,27 @@
 
 
 @dataclass
-class AppConfig:
-    name: str = field(default="mini-wallet")
+class ServerConfig:
     url_scheme: str = field(default="http")
     host: str = field(default="localhost")
```

Comment:
localhost -> 0.0.0.0

Reply:
specifically avoided `0.0.0.0` here because it breaks the self test